.github/workflows/airbyte-ci-tests.yml

```diff
@@ -30,6 +30,7 @@ jobs:
         with:
           # Note: expressions within a filter are OR'ed
           filters: |
+            # This list is duplicated in `pipelines/airbyte_ci/test/__init__.py`
```

Comment:
This was just a note for me when I was working on the [upstack PR](https://github.com/airbytehq/airbyte/actions/runs/8446603559/job/23135651978?pr=36497)

There is some weird stuff around keepign this list up to date and also the output of changes being calculated differently from the `--modified` flag that we should look into later but I haven't addressed in this stack.